packages/destination-actions/src/destinations/blackbaud-raisers-edge-nxt/createConstituentAction/index.ts

```diff
@@ -0,0 +1,165 @@
+import { ActionDefinition, ExecuteInput, InputField, IntegrationError, RequestFn } from '@segment/actions-core'
+import type { Settings } from '../generated-types'
+import type { Payload } from '../createConstituentAction/generated-types'
+import { perform as performCreateOrUpdateIndividualConstituent } from '../createOrUpdateIndividualConstituent'
+import { BlackbaudSkyApi } from '../api'
+import { ConstituentAction, StringIndexedObject } from '../types'
+import {
+  augmentFieldsWithConstituentFields,
+  buildConstituentActionDataFromPayload,
+  buildConstituentPayloadFromPayload
+} from '../utils'
+
+const fields: Record<string, InputField> = augmentFieldsWithConstituentFields({
+  date: {
+    label: 'Date',
+    description: 'The action date in ISO-8601 format.',
+    type: 'datetime',
+    required: true,
+    default: {
+      '@path': '$.timestamp'
+    }
+  },
+  category: {
+    label: 'Category',
+    description:
+      'The channel or intent of the constituent interaction. Available values are Phone Call, Meeting, Mailing, Email, and Task/Other.',
```

Comment:
I feel like for the fields where we're including `choices`, we don't need to list the available values in the description.